requirements.txt

```diff
@@ -1,4 +1,7 @@
 pydicom
 # This is a dcmstack branch with changes for Python 3
-git+git://github.com/ghisvail/dcmstack@master
+git+git://github.com/mvdoc/dcmstack@bf/importsys
```

Comment:
is there a PR you have submitted with some fixes?  Please add a comment here so we could later check if it was merged and eventually switch to the released dcmstack